classes/controller/FrontController.php

```diff
@@ -1698,6 +1698,7 @@ public function getTemplateVarShop()
 
         $shop = [
             'id' => $this->context->shop->id,
+            'id_shop_group' => $this->context->shop->id_shop_group,
```

Comment:
```suggestion
            'group_id' => $this->context->shop->id_shop_group,
```

Just a suggestion, to me`$shop.id` and `$shop.group_id` look more consistent.